### PR TITLE
Feature/investment path fixup

### PR DESCRIPTION
**Pull request by jim68000**

### CODE REVIEW

File: generator/specs/definitions/investment_defs.yml [hunk 1]
```diff
@@ -70,7 +70,9 @@
       sector:
         $ref: '#/definitions/Sector'
       business_activity:
```
Comment (teneightfive) [inline generator/specs/definitions/investment_defs.yml]: Should this be a plural if it's an array?

Comment (reupen) [inline generator/specs/definitions/investment_defs.yml]: Rename to business_activities?

File: generator/specs/definitions/investment_defs.yml [hunk 2]
```diff
@@ -318,7 +320,6 @@
         type: string
         example: 'Active'
   StrategicDriverArray:
-    type: array
```
Comment (reupen) [inline generator/specs/definitions/investment_defs.yml]: What was the reason for this change?

Comment (jim68000) [inline generator/specs/definitions/investment_defs.yml]: Weird, I don't remember deleting this. Restored.

File: generator/specs/definitions/investment_defs.yml (hunk 1 above)
Comment (jim68000) [inline generator/specs/definitions/investment_defs.yml]: Done

File: generator/specs/paths/investment.yml [hunk 3]
```diff
@@ -1,4 +1,39 @@
   /v3/investment/project:
+    get:
+      tags:
+      - investment
+      summary: returns a summary of investment projects for a company
+      operationId: getprojectsforcompany
+      description: Returns a summary list of investment projects per investor-company ID
+      consumes:
+      - application/json
```
Comment (reupen) [inline generator/specs/paths/investment.yml]: Not too important, but this doesn't seem relevant as it's a GET operation.

**Pull request merged**